File: solutions/beecrowd/1794/1794.cpp
#include <cstdint>
#include <iostream>

int main() {
    std::int16_t n, la, lb, sa, sb;

    while (std::cin >> n >> la >> lb >> sa >> sb) {
        if (n >= la && n <= lb && n >= sa && n <= sb) {
            std::cout << "possivel";
        } else {
            std::cout << "impossivel";
        }
        std::cout << std::endl;
    }

    return 0;
}

File: solutions/beecrowd/1794/1794.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    n = int(line)
    la, lb = map(int, input().split())
    sa, sb = map(int, input().split())

    if n >= la and n <= lb and n >= sa and n <= sb:
        output('possivel\n')
    else:
        output('impossivel\n')
